test/platform/profiles/profile.spec.ts

```diff
@@ -2,9 +2,27 @@ import expect from 'expect'
 import { Profile } from '../../../src'
 import { testTypeSignature } from '../../test-utils'
 import { AVATAR } from './avatar.spec'
+import { Country } from '../../../src/platform/profile/additional-info/country'
+import { EmploymentStatus } from '../../../src/platform/profile/additional-info/employment'
+import { Gender } from '../../../src/platform/profile/additional-info/gender'
+import { RelationshipStatus } from '../../../src/platform/profile/additional-info/relationship'
+import { SexualOrientation } from '../../../src/platform/profile/additional-info/sexual-orientation'
+import { Language } from '../../../src/platform/profile/additional-info/language'
 
 describe('Profile tests', () => {
-  const profile = { avatars: [AVATAR] }
+  const profile = {
+    avatars: [AVATAR],
+    country: Country.ARGENTINA,
+    employment_status: EmploymentStatus.CHILLING,
+    gender: Gender.FEMALE,
+    relationship_status: RelationshipStatus.COMPLICATED,
```

Comment:
Don't forget to camelCase these... perhaps add better tests, so that, in case of wrong values, we can have an idea of the errors we're reporting.